Creates README.md

elainemattos opened the pull request:
thats it, thats the PR.

elainemattos committed:
Create README.md

FelipeAlves99 commented on README.md:
```diff
@@ -0,0 +1,3 @@
+# APS_6_Semestre
+
+Supervised practical activities for the 5th semester of the Computer Science College.
```
It's the 6th semester...

elainemattos committed:
Fix README.md

as hannah montana once said "everybody makes mistakes everybody has those days"
